src/main/scala/org/allenai/plugins/CoreDependencies.scala

```diff
@@ -88,6 +88,14 @@ trait CoreDependencies {
   lazy val allenAiTestkit = allenAiCommonModule("testkit")
   lazy val allenAiWebapp = allenAiCommonModule("webapp")
   lazy val allenAiIndexing = allenAiCommonModule("indexing")
+  lazy val allenAiCache = {
+    val splitVersion = allenAiCommonVersion.split('.')
```

Comment:
Aside: I hate this method in Scala's enhanced string class, because it looks like it's not using a regex under the covers, but it totally is. Don't use this in critical code sections. :)